locust/main.py

```diff
@@ -469,6 +498,9 @@ def main():
         if not options.no_web:
             logger.error("The --run-time argument can only be used together with --no-web")
             sys.exit(1)
+        if options.slave:
```

Comment:
I think this change is not really related to the issue you are solving? Currently I rely on slaves accepting (and ignoring) this parameter, so I'd prefer you not to change this :)